### PR TITLE
modif testimonyStatus for validate testimony

### PR DESCRIPTION
**Pull request by Vannou28**

### CODE REVIEW

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

File: src/Model/TestimoniesManager.php [hunk 1]
```diff
@@ -23,19 +23,31 @@ public function insert(array $testimony): int
...
         $statement = $this->pdo->prepare($query);
-        $statement->bindValue('name', $item['name'], \PDO::PARAM_STR);
-        $statement->bindValue('mail', $item['mail'], \PDO::PARAM_STR);
-        $statement->bindValue('message', $item['message'], \PDO::PARAM_STR);
-        $statement->bindValue('validation', $item['validation'], \PDO::PARAM_BOOL);
+        $statement->bindValue('name', $testimony['name'], \PDO::PARAM_STR);
+        $statement->bindValue('mail', $testimony['mail'], \PDO::PARAM_STR);
+        $statement->bindValue('message', $testimony['message'], \PDO::PARAM_STR);
+        $statement->bindValue('validation', $testimony['validation'], \PDO::PARAM_BOOL);
+        $statement->bindValue('id', $testimony['id'], \PDO::PARAM_INT);
```
Comment (LucyMerlier) [inline src/Model/TestimoniesManager.php:38]: do you really need a method to update a whole testimony? wouldn't it be problematic if Nathalie could twist what her clients are saying about her?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**